### PR TITLE
[stable/gcloud-sqlproxy] Add PDB

### PR DESCRIPTION
**Pull request by coreypobrien**
Adds a PDB for high availability

### CODE REVIEW

**Comment by coreypobrien** [comment]
@unguiculus Any reason not to move forward on this?

**Comment by coreypobrien** [comment]
@rimusz @lachie83 Any feedback on this?

**Comment by coreypobrien** [comment]
Caught up with chart outdated chart version and signed off on old commit. Should be good to go now.

**Comment by rimusz** [comment]
yes, looks good now

File: stable/gcloud-sqlproxy/templates/pdb.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  labels:
+    app: {{ template "gcloud-sqlproxy.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "gcloud-sqlproxy.fullname" . }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "gcloud-sqlproxy.name" . }}
```
Comment (unguiculus) [inline stable/gcloud-sqlproxy/templates/pdb.yaml:14]: The selector needs the `release` label as well.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/gcloud-sqlproxy/templates/pdb.yaml (hunk 1 above)
Comment (coreypobrien) [reply stable/gcloud-sqlproxy/templates/pdb.yaml:14]: Just to be clear, do you want me to change the `Deployment` too? the current pod spec doesn't include the `release` label afaict:

```
  template:
    metadata:
      labels:
        app: {{ template "gcloud-sqlproxy.fullname" . }}
    spec:
```

**Pull request merged**